pkg/genericapiserver/config.go

```diff
@@ -282,6 +284,7 @@ func (c *Config) ApplyOptions(options *options.ServerRunOptions) *Config {
 				},
 			},
 			SNICerts: []NamedCertKey{},
+			ServerCA: options.TLSCAFile,
```

Comment:
Wondering, is that ca concatenated to the cert used anywhere? The https server in the apiserver does not need it.